gpsr/helpers.go

```diff
@@ -69,3 +69,11 @@ func appendToGraph(m Graph, d desc.Desc, dependencyConfigs InstallDeps, pkgNexus
 		}
 	}
 }
+
+
+// Left off here -- we can easily gather a list of Dep objects and try to append them,
+// but if we add the full-blown Desc objects, then we'll be putting the tarball package in the install plan.
+// We're probably just going to do a hacky workaround for this.
+func appendSingleDependencyToGraph(m Graph, d desc.Dep, pkgNexus *cran.PkgNexus) {
```

Comment:
this function scaffold has two things that could be improved - first dependency is already singular - so could be appendDependencyToGraph(...). Furthermore, this could easily be generalized to take a variadic param of desc's, so could easily pass those one or many

```
appendDependenciesToGraph(m, pkgNexus, d)
appendDependenciesToGraph(m, pkgNexus, d1, d2, d3)
appendDependenciesToGraph(m, pkgNexus, listOfDeps...)
```